erpnext_documentation/www/docs/user/manual/en/accounts/process-statement-of-accounts.md

```diff
@@ -0,0 +1,102 @@
+
+    - "From Date" and "To Date" filters will be hidden and auto-filled dynamically when "Enable Auto Email" option is selected.
+    - "Project" and "Cost Center" fields are multi-select.
+
+4. In the "Customers" section you have an option to select customers in the child table and fetch their primary and billing emails. 
+
+    - The "Select Customer By" select field lets you select customers in bulk, by grouping them based on "Customer Group", "Territory", "Sales Partner" and "Sales Person" by entering the selection and clicking on "Fetch Customers". 
+    - In tree doctypes like "Territory", "Sales Person", and "Customer Group" on selecting group values, the customers having the child values of these fields will also be fetched. So when you select "India" as territory in the form, all customers with "Territory" values under India in the Territory tree will get selected.
+    - The "Send To Primary Contact" option will send the Statement Of Account to the primary contact email-ids of the customers too apart from the billing email.
+
+    <img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/customers.png"><br>
+
+5. In "Print Preferences" section you can select 2 things:
+
+    - Print orientation of the PDF file, either "Landscape" or "Portrait".
+    - Whether you want to see the ageing report (Accounts Receivable Summary report), which shows the ageing amount for 30/60/90/120 days for vouchers (like Sales Invoice), based on either "Due Date" or "Posting Date".
+
+    <img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/print.png">
+
+6. The "Email Settings" section lets you configure how you want the emails to be sent. There are two subsections in this:
+
+    <img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/auto-email.png">
+    
+    - On selecting "Enable Auto Email" you will see the options to send automated periodic reports to the customers in the entry.
+
+        - You can select the "Frequency" at which the emails will be sent after the "Start Date" to the customers.
+        - You can also select the "Filter Duration", which is the gap between the "From Date" and "To Date" values in the General Ledger report until the date of sending the report ("To Date").
+        - These mails are not sent right away, but at midnight as a background process.
+    
+    - After this you can select the "Subject", "CC To" and "Body" fields of the email. If you don't set values to this field, default values will be used like shown in the help box below the body field.
+    
+    <img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/email-content.png">
+
+7. Review your settings and click on "Save"
+
+## 3. Features
+
+<img class="screenshot" src="{{docs_base_url}}/assets/img/accounts/process-statement-of-accounts/buttons.png">
```

Comment:
image later not before

Reply:
Added becuase the 1st two points needs its reference.
Will move it to after the first 2 points.